Reverse transcription quantitative real-time PCR (RT-qPCR) is known for its high accuracy and specificity and is currently the most reliable method for detecting gene expression. The choice of reference genes significantly impacts the accuracy of target gene expression results. However, selecting reliable reference genes for specific tissue types or growth conditions in rice remains limited. Here, we present the Rice Reference Genes (RRG) tool (http://www.rrgenes.com/), which addresses some limitations in selecting reference genes for various experimental conditions in rice. This website is generated by mining extensive rice RNA-seq datasets, comprising a total of 4,404 RNA-seq datasets. These are categorized into five different tissues—leaf, root, seedling, panicle, and seed—and seven types of stress conditions (cold, disease, drought, heat, hormone, metal, and salt) along with their respective controls (mock). In this study, we also utilized RRG to screen candidate reference genes in rice leaves, roots, and seedlings under salt and drought stress. We rigorously evaluated the suitability of these candidate reference genes against commonly used established reference genes, confirming the accuracy and applicability of the reference genes selected through RRG. This tool opens an accessible door to even inexperienced user for choosing the right reference genes in rice with proper flexibility.
